arlo-client/src/components/AuditForms/CalculateRiskMeasurement.tsx

```diff
@@ -177,6 +202,48 @@ const CalculateRiskMeasurement: React.FC<IProps> = ({
     }
   }
 
+  const downloadDataEntry = async (): Promise<void> => {
```

Comment:
Nothing in this function actually requires it be `async`. Is there a missing `await` somewhere?

Reply:
@eventualbuddha Good catch, I was copying over from the other pdf generation functions, and those used the `await getBallots()` function, but this doesn't.